components/notifier/src/quality_time_notifier.py

```diff
@@ -36,7 +37,7 @@ async def notify(log_level: int = None) -> NoReturn:
         except Exception as reason:  # pylint: disable=broad-except
             logging.error("Could not get reports from %s: %s", reports_url, reason)
             json = dict(reports=[])
-        notifications = get_notable_metrics_from_json(data_model, json, most_recent_measurement_seen)
+        notifications = notable.get_notable_metrics_from_json(json, most_recent_measurement_seen)
```

Comment:
The method is called "get_notable_metrics_from_json" but the result is named "notifications". They can't both be correct. Please rename either the method or the variable